src/Client/KeenIOClient.php

```diff
@@ -90,15 +90,7 @@ function($arg)
      */
     public function __call($method, array $args)
     {
-        if (isset($args[0]) && is_string($args[0])) {
-            $args[0] = array('event_collection' => $args[0]);
-
-            if (isset($args[1]) && is_array($args[1])) {
-                $args[0] = array_merge($args[1], $args[0]);
-            }
-        }
-
-        return parent::__call($method, isset($args[0]) ? $args[0] : array());
+        return parent::__call($method, array($this->combineEventCollectionArgs($args)));
```

Comment:
I'll take this one back - you say it requires an array of arrays. Ignore this one :)